packages/period-selector-dialog/src/reducers/__tests__/periods.spec.js

```diff
@@ -0,0 +1,161 @@
+import { periods } from '../periods';
```

Comment:
It should be possible to import these 2 first lines in a single statement (same with periodType.spec.js file)